src/components/NatureOfBusiness/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@bcrs-shared-components/nature-of-business",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@bcrs-shared-components/interfaces": "^1.0.31",
```

Comment:
I'm hoping this version of the dependency contains my new interfaces, or that Lerna will auto-update it. We'll see.

(Locally, ~/src/components/NatureOfBusiness/node_modules/@bcrs-shared-components/interfaces/ contains my new interfaces, and I didn't copy them there. 🤷‍♂️ )

Reply:
Is this the future version or what it is now? 
I imagine you could put what it is now, and Lerna will update this? 
I could be wrong on this tho..

Reply:
This is the now version.